integrations/huggingface-transformers/tutorials/token-classification/token-classification-cli.md

```diff
@@ -0,0 +1,273 @@
+modifiers, by setting `--distill_teacher disable` we instruct SparseML to skip distillation.
+
+### WNUT_17 Inspection
+
+Run the following to inspect the Rotten Tomatoes dataset.
+
+```python
+from datasets import load_dataset
+
+wnut_17 = load_dataset("wnut_17")
+print(wnut_17)
+print(wnut_17["train"][0])
+```
+
+We can see that each row dataset contains a `tokens` field which contains a list of strings representing each word the sentence and a corresponding `ner_tags` which is a list of integers representing the tag of each word in the sentence.
+
+### Using a Hugging Face Dataset Identifier
+
+To use this dataset with the CLI, we can replace the `--dataset_name conll2003` argument with `--dataset_name wnut_17 --input_column_names tokens --label_column_name ner_tags`. SparseML will then download the dataset from the Hugging Face hub and run training as before.
+
+```
+sparseml.transformers.token_classification \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --recipe zoo:nlp/token_classification/obert-base/pytorch/huggingface/conll2003/pruned90_quant-none \
+  --recipe_args '{"num_epochs":12,"qat_start_epoch":7.0, "observer_epoch": 11.0}' \
+  --distill_teacher disable \
+  --dataset_name wnut_17 --text_column_name tokens --label_column_name ner_tags \
+  --output_dir sparse_bert-token_classification_wnut_17 \
+  --per_device_train_batch_size 32 --per_device_eval_batch_size 32 --preprocessing_num_workers 6 \
+  --do_train --do_eval --evaluation_strategy epoch --fp16 --seed 29204  \
+  --save_strategy epoch --save_total_limit 1
+```
+
+You will notice that we used the same recipe as we did in the Conll2003 case (identified by the SparseZoo stub `zoo:nlp/token_classification/obert-base/pytorch/huggingface/conll2003/pruned90_quant-none`). Since the WNUT and Conll2003 tasks are similiar, we chose to start with the same hyperparameters as we used in Conll2003 training.
+
+#### Recipe Args
+
+To update a recipe and experiment with hyperparameters, you can download the YAML file from SparseZoo, make updates to the YAML directly, and pass the local path to SparseML.
+
+Alternatively, you can use `--recipe_args` to modify the recipe on the fly. In this case, we used the following to run for 12 epochs instead of 13:
+
+```bash
+--recipe_args '{"num_epochs":12,"qat_start_epoch":7.0, "observer_epoch": 11.0}'
+```
+
+### Using Local CSV/JSON Files
+
+Let's walk through how to pass a CSV/JSON dataset to the CLI.
+
+#### Save Dataset as a CSV File
+
+For this example, we use Hugging Face `datasets` to create a JSON file for WNUT_17 that can be passed to SparseML's CLI but you can use any framework you want. For the Token Classification CLI, the label column must contain actual tags (i.e. not indexes). As such, we need to map the NER ids to tags before saving to JSON.
```

Comment:
For this example, we use Hugging Face `datasets` to create a JSON file for WNUT_17 that can be passed to SparseML's CLI, but you can use any framework you want.